content/java/concepts/math-methods/terms/pow/pow.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '.pow()'
+Description: 'Returns the first argument raised to the power of the second argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.pow()`** method returns the first argument raised to the power of the second argument.
+
+## Syntax
+
+```pseudo
+Math.pow(b, p)
+```
+
+`b` and `p` are both `double` values.
```

Comment:
```suggestion
The `base` and `exponent` parameters are both `double` values.
```

Can we also include a brief list that describes some common edge cases. For example, what would the return value for `.pow()` be if `base` were `NaN`, or if `exponent` were equal to 0.0 or 1.0?